src/libsyntax/errors/mod.rs

```diff
@@ -38,10 +38,6 @@ pub enum RenderSpan {
     /// of hypothetical source code, where each `String` is spliced
     /// into the lines in place of the code covered by each span.
     Suggestion(CodeSuggestion),
```

Comment:
Can we just get rid of RenderSpan? Suggestion seems different enough that it can just be a standalone struct at this point. And RenderSpan just bothers me.